python_modules/libraries/dagster-airbyte/dagster_airbyte/asset_defs.py

```diff
@@ -82,14 +82,14 @@ def _build_airbyte_asset_defn_metadata(
 
     internal_deps: Dict[str, Set[AssetKey]] = {}
 
-    normalization_tables = (
+    lst_normalization_tables = (
```

Comment:
nitpick: avoid abbreviations like "lst". it seems like the reason we're doing this is to enable including it as metadata? if so, I think a good name would be something like `metadata_encodable_normalization_tables`

Reply:
ok, renamed